spec/models/media_attachment_spec.rb

```diff
@@ -295,8 +295,8 @@
     end
 
     it 'queues CacheBusterWorker jobs' do
-      original_path = media.file.path(:original)
-      small_path = media.file.path(:small)
+      original_path = media.file.url(:original)
+      small_path = media.file.url(:small)
```

Comment:
I would maybe rename the variables here (`_url` instead of `_path`) to make it super clear what is expected.